std/experimental/allocator/mallocator.d

```diff
@@ -64,19 +64,19 @@ struct Mallocator
 }
 
 ///
-unittest
+@nogc unittest
 {
     auto buffer = Mallocator.instance.allocate(1024 * 1024 * 4);
     scope(exit) Mallocator.instance.deallocate(buffer);
     //...
 }
 
-unittest
+@nogc unittest
 {
-    static void test(A)()
+    @nogc static void test(A)()
     {
         int* p = null;
-        p = new int;
+        p = cast(int*) A.instance.allocate(int.sizeof);
```

Comment:
The leak doesn't matter, it'a test, so single shot program. However I can add the deallocation code if you wish to. Tell me clearly yes / no.

And yes, the test funct was initially nothing bu useless, I just modified it to get an additional @nogc checking.